ovirt-hosted-engine-setup.spec.in

```diff
@@ -72,14 +72,9 @@ Requires:       sudo
 Requires:       libvirt-client >= 6.6.0-9
 Requires:       virt-install
 
-Requires:       ansible-core >= 2.12
-# ansible-core 2.13 uses Python 3.9, and many modules are missing a Python 3.9
-# build on el8. TODO: Provide them? Instead of or in addition to Python 3.8?
-%if 0%{?rhel} < 9
-Conflicts:      ansible-core >= 2.13
-%endif
+Requires:       ansible-core >= 2.13
```

Comment:
I would consider adding a conflict on >=2.14 as per issue #68

Reply:
No, we need to be compatible with 2.14 unless we want to support Rocky/Alma 8.8 and 9.2.
The good thing is that both 2.13 and 2.14 are using Python 3.9, so hopefully not big changes needed from us

Reply:
Unless I'm missing something https://github.com/oVirt/ovirt-hosted-engine-setup/issues/68 should be fixed by https://github.com/oVirt/ovirt-hosted-engine-setup/pull/66

Reply:
great, thanks